tests/test_epic_dist.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
```

Comment:
We could use `hypothesis` to make these tests more robust. You can checkout my tests in https://github.com/HumanCompatibleAI/tabular-baseline/blob/4d32afd3ba30f6eb50e9203735479a2a725e96df/tests/test_epic_distance.py